css-brochure-form/styles.css

```diff
@@ -0,0 +1,60 @@
+h2 {
+  text-align: center;
+  font-family: "Brandon Grotesque", sans-serif;
+  text-transform: uppercase;
+  color: rgb(184 144 58);
```

Comment:
> <img alt="Screen Shot 2022-03-16 at 3 32 02 PM" width="1672" src="https://user-images.githubusercontent.com/20695764/158702330-afcc8542-1b7e-4747-886c-b96384d12c15.png">
> 
> I tried my best to get it to a more orange-red color. I'm red-green color blind lol

In that case, there are a couple ways you can deal with issues like this.
- In a professional environment, the first thing you can try is to have the client or designer (if there is one) provide you with a color code.
- If you have to make the color more-red, you can take the `rgb` value and move some of the second number into the first number (vice versa if it needs to be more green).
- If you happen to have an in-browser example to pull from, you can use the development tools' color picker to obtain the color code from the example:

---

![image](https://user-images.githubusercontent.com/16143771/158704962-d338e59f-d88c-48af-926c-4854e694a074.png)